server/src/internalClusterTest/java/org/opensearch/action/admin/indices/create/RemoteSplitIndexIT.java

```diff
@@ -89,12 +93,32 @@
 
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class RemoteSplitIndexIT extends RemoteStoreBaseIntegTestCase {
+    @Before
+    public void setup() {
+        asyncUploadMockFsRepo = false;
+    }
 
     @Override
     protected boolean forbidPrivateIndexSettings() {
         return false;
     }
 
+    @After
+    public void cleanUp() throws Exception {
```

Comment:
rename to teardown? to be in parity with base class?

Reply:
We can't do that as it uses annotation. It throws an error on doing that .